whatYouNeedToKnow.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "What You Need To Know"
+ready: True 
+---
+
+
+# What You Need To Know
+
+This is a guide to the Tech Department and how it works. It contains contact details, important links to documents we use and software that are useful. 
+
+  
+
+## Daily Tech Events.
+
+These happen everyday and are compulsory for everyone to attend.
+
+##### Stand up 
+This is how we begin our day, announcements are shared and we discuss anything that hinders progress. 
+ - Every morning at 09:00
+
+##### Lunchtime presentation
+*handled by Kii*
+This is where we exercise our presentation muscle,  one recruit prepares a 5 minute  presentation about something tech related - what you’ve learnt, something you’re working on or any resources that you think might be useful. There is a schedule that specifies who will present on which day.   
+- Everyday after lunch at 13:30, 
+
+##### The Productivity Award
+*Handled by Kii*
+To encourage productivity, recruits are allocated points on a specific Criteria which comprises:
+- Attendance 
+- Project Submission
+- Posting relevant material on Slack
+- Asking and/or providing solutions to a problem posted on Slack
+- Presentations
+- Reviewing someone else’s code
+- Doing useful pull requests
+- Having your code merged.
+
+
+
+##### Tech Meetups
+*Handled by Singi*
+
+ At least once a month (usually more)  we attend a meetup. We get to learn on what's new in the tech world, learn more on the technlogy and languages we already know. 
+ 
+
+
+
+## Communication
+
+Communication is a crucial part of the department and also encourages productivity.
+We use:
+##### Emails
+For any official communication and setting up meetings, events and workshops.
+
+##### WhatsApp groups
+For communication within cohorts and the mighty Coding Department WhatsApp group for the whole department.
+
+##### Slack channels
+There are various slack channels for different purposes. The sole purpose is to administer communication between recruits and the tech staff:
+
+- To get feedback from managers.
+- To share relevant material and opportunities with fellow recruits.
+- To get departmental and cohort specific announcements.
+
+
+If you have any queries, these are the people to contact:
+### Junior Tech Facilitators 
+**Web Development:**
+
+Mhlanganisi Kii Gxobole - mhlanganisi.gxobole@umuzi.org
+Tshepo Modise - tshepo.modise@umuzi.org
+
+**Data:**
+Singita Ngobeni - singita.ngobeni@umuzi.org
+
+### Senior Managers
+**Web Development:** 
+Sheena O’connell - sheena.oconnell@umuzi.org
+Dibwe Kalangu - dibwe.kalangu@umuzi.org
+
+**Data:**
+Michelle Hoogenhout - michelle.hoogenhout@umuzi.org
+
+
+### Important Spreadsheets and Forms (I suggest you bookmark these)
+- Coding & Data Programme Outline - https://docs.google.com/spreadsheets/d/14SsiRw8sit3-IvzpntINicIWd4MG1CDOxbv14Ypsmpw/edit?usp=sharing
```

Comment:
This spreadsheet should be considered the source of truth about any upcomming deadlines so please pay close attention. The tech staff updates this spreadsheet once per week, usually on a Monday afternoon.

Reply:
Also link to the relevant calendars.